docs/participate/online-communities.mdx

```diff
@@ -8,46 +8,34 @@ The Mina community has many online groups where you can connect with other Mina
+[Mina Protocol Discord](https://discord.gg/minaprotocol) is the most popular place where Mina enthusiasts and technical contributors gather. Some popular channels are listed here, but many more channels exist, so explore the server to find one that fits your interest.
 
-[#announcements](https://discord.gg/JRU3PENEB2) — to follow announcements
-
-[#zkapps-general](https://discord.gg/MVfuuEvD4U) — to ask general questions about zkApps, how to use a zkApp, etc.
-
-[#zkapps-developers](https://discord.gg/v3W4YsuBCw) — to meet other developers building zkApps with SnarkyJS
-
-[#mentor-nodes](https://discord.gg/j9GHr7pJrR) — to get help from the community on setting up and running nodes
-
-[#mainnet-block-producer](https://discord.gg/eAD6V4wW7N) — to meet other node operators and block producers
-
-[#grants](https://discord.gg/AcFCvVvfVW) — to discuss grant opportunities
+- **#announcements** to follow announcements
+- **#zkapps-general** to ask general questions about zkApps, how to use a zkApp, and so on
+- **#zkapps-developers** to meet other developers building zkApps with SnarkyJS
+- **#mentor-nodes** to get help from the community on setting up and running nodes
+- **#mainnet-block-producer** to meet other node operators and block producers
+- **#grants** to discuss grant opportunities
 
 ## Discussion Forums
 
-[Mina Research Forums](https://forums.minaprotocol.com/)
-
-[Mina on Github Discussions](https://github.com/MinaProtocol/mina/discussions) — to discuss new Mina releases
+- [MinaResearch](https://forums.minaprotocol.com/)
+- [MinaProtocol/mina Github](https://github.com/MinaProtocol/mina/discussions)
 
 ## Social Media
 
-[Twitter - @minaprotocol](https://twitter.com/minaprotocol)
-
-[Telegram - @minaprotocol](https://t.me/minaprotocol)
-
-[Telegram - @MinaProtocolOfficialNews (Announcements Only)](https://t.me/MinaProtocolOfficialNews)
-
-[Reddit - /r/minaprotocol](https://www.reddit.com/r/MinaProtocol/)
+- Twitter [@minaprotocol](https://twitter.com/minaprotocol)
+- Telegram [@minaprotocol](https://t.me/minaprotocol)
+- Telegram [@MinaProtocolOfficialNews](https://t.me/MinaProtocolOfficialNews)
+- Reddit [/r/minaprotocol](https://www.reddit.com/r/MinaProtocol/)
 
 ## YouTube
 
-[YouTube - Mina Foundation](https://www.youtube.com/c/minaprotocol)
-
-[YouTube - Evan Shapiro, CEO of Mina Foundation](https://www.youtube.com/channel/UCZebnzqd__m_lHP_d5qgsoQ)
-
-[YouTube - O(1) Labs](https://www.youtube.com/channel/UCrT2KUIIrDPJk8leVu7GYdQ)
+- [Mina Foundation](https://www.youtube.com/c/minaprotocol)
+- [Evan Shapiro, CEO of Mina Foundation](https://www.youtube.com/channel/UCZebnzqd__m_lHP_d5qgsoQ)
+- [O(1) Labs](https://www.youtube.com/channel/UCrT2KUIIrDPJk8leVu7GYdQ)
```

Comment:
oooh now that YouTube URLs rely on user handles, we can eventually grab the right [handle](https://support.google.com/youtube/answer/6180214?hl=en)

Reply:
We can use https://www.youtube.com/@o1labsofficial now right? It only recently crossed the 100 subs threshold where Regina was able to set up the custom URL.